### PR TITLE
Created separe class file

### PR DESCRIPTION
**Pull request by akstrfn**
I've created a separate class file and did some other modifications. I will work on it some more but for now you can let me know what you think about it.

### CODE REVIEW

**Comment by chiehrosswang** [comment]
Thanks for the changes, I haven't had time to go through all the changes you made but they look great to me!

**Comment by akstrfn** [comment]
I think you can pull this one if you'd like. I will probably add stuff from time to time to the template since it's I use it as a submodule and add things from time to time but the template is usable now. :smile:

File: trb_template.tex [hunk 1]
```diff
@@ -1,365 +1,212 @@
...
-\bibliographystyle{trb}
-\bibliography{trb_template}
-
-% End line numbering
-\nolinenumbers
-\end{document}
+% When numbered option is activated, lines are numbered.
+\documentclass[numbered]{trbunofficial}
+\usepackage{graphicx}
+\usepackage[colorlinks=true,linkcolor=blue,citecolor=blue]{hyperref}
```
Comment (chiehrosswang) [inline trb_template.tex]: I'd use `\usepackage[hidelinks]{hyperref}` to keep it closer to the author guidelines and the formatting model provided by TRB (no color for hyperlinks and citations).

File: trb_template.tex [hunk 2]
```diff
@@ -1,365 +1,212 @@
...
+
+\section{Introduction}
+The \trbcite{TRBGuide} has unique and somewhat arbitrary requirements for
+papers submitted for review and publication. While the initial submission is
+required to be in PDF format, submissions for publication in Transportation
+Research Record must be in Microsoft Office format. On top of this, the
+manuscripts must be line-numbered, captions are bolded and employ atypical
+punctuation, and the references must be numbered when cited and then printed in
+order. More details about the manuscript details can be found
+online~\footnote{\url{http://onlinepubs.trb.org/onlinepubs/AM/InfoForAuthors.pdf}}.
```
Comment (chiehrosswang) [inline trb_template.tex]: TRB does not allow footnotes.  I'd change it to: `online at~\url{http://onlinepubs.trb.org/onlinepubs/AM/InfoForAuthors.pdf}.`

File: trb_template.tex [hunk 3]
```diff
@@ -1,365 +1,212 @@
...
+written by \trbcite{Garrow2009} and \trbcite{Koppelman2005}. You can even go
+back and refer to Biog\'eme by \trbcite{Bierlaire2008} a second time. You can
+also cite a group of similar references without printing author names
+\cite{TRBGuide,Bierlaire2003}. This template also groups multiple reference
+numbers together if there are three or more consecutive numbers
+\cite{Bierlaire2003,Bierlaire2008,Garrow2009,Koppelman2005}.
+
+\subsection{Equations}
+Intelligent driver model equations from wikipedia
+\url{https://en.wikipedia.org/wiki/Intelligent_driver_model} moved to the right
```
Comment (chiehrosswang) [inline trb_template.tex]: Do you mean flush to the left?

File: trb_template.tex [hunk 4]
```diff
@@ -1,365 +1,212 @@
...
+  \caption{A History of this Template}\label{tab:versions}
+  \begin{center}
+    \begin{tabular}{l l l l}
+Version & Date & Author & Contributions \\\hline
+1.0   & Sep 2009 & Pritchard & Initial work \\
+1.1   & Mar 2011 & Pritchard & Captions \\
+2.0   & Mar 2012 & Macfarlane& Automation, documentation\\
+2.1   & Jul 2015 & Wang      & More automation and formatting\\
+2.1.1 & Jan 2016 & Wang      & Minor modifications and uploaded to Github\\
+2.1.1 Lite & Jun 2017 & Wang & TeX-only template \\hline
```
Comment (chiehrosswang) [inline trb_template.tex]: Should be `\\\hline`

**Comment by chiehrosswang** [comment]
I've commented some minor stuff but overall it looks great (and clean)!  Please see my inline comments and let me know what you think.

**Comment by akstrfn** [comment]
Thanks for the comments @chiehrosswang! Template is updated. Only think that I kept are the coloured links but I added a comment that it must be removed to final TRB submission. That should probably also be mentioned somewhere in the text too.

**Pull request merged**